docker-assets/initialize-httpd-auth.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+HTTPD_AUTH_CONFIG_DIR=/etc/httpd/auth-conf.d
+AUTH_CONFIG_FILE="${HTTPD_AUTH_CONFIG_DIR}/auth-configuration.conf"
+SSSD_CONF_FILE="/etc/sssd/sssd.conf"
```

Comment:
@abellotti To dump all environment to a file before systemd gets in, we can use a kube postStart lifecycle hook on the httpd DC, it takes a command or a script/entrypoint. 

We already use some lifecycle hooks across the deployment, it makes sense for something simple like this. Consider the following gist : 

https://gist.github.com/fbladilo/83d53f104985e5d48debae58ab6b5e25